docker/nvidia/setup_nvidia_docker_compose.py

```diff
@@ -120,19 +120,7 @@ def get(self, endpoint):
     config['services'][service]['devices'] = devices
     config['services'][service]['volumes'] = volumes
     config['services'][service]['environment'] = ['GPU_ARCH=' + gpu_arch]
-    config['services'][service]['image'] = service + '_cuda_toolchain'
-    dockerfile = 'Dockerfile_' + service + '_cuda_toochain_generated'
-    config['services'][service]['build'] = {
-        'context': '.',
-        'dockerfile': dockerfile,
-        'args': [
-            'CUDA_VERSION=' + cuda_version,
-            'CUDA_PKG_VERSION=' + cuda_pkg_version,
```

Comment:
I think that `cuda_pkg_version` is not used anymore. So we can remove all the code associated with it.

Reply:
You are correct.  I will address that in another PR when we move towards nvidia-docker 2.0 if that's OK with you.